Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3173,18 +3216,241 @@ def fetch_incidents():
             current_fetch_ofp_detection,
         ]
     )
-    return (
-        incidents
+    return (incidents
         + detections
         + idp_detections
         + iom_incidents
         + ioa_incidents
         + mobile_detections
         + on_demand_detections
-        + ofp_detections
-    )
+        + ofp_detections)
+
+def fetch_events():
+    incidents: list = []
+    detections: list = []
+    idp_detections: list = []
+    mobile_detections: list[dict[str, Any]] = []
+    on_demand_detections: list[dict[str, Any]] = []
+    ofp_detections: list[dict[str, Any]] = []
+    last_run = demisto.getLastRun()
+    demisto.debug(f"CrowdStrikeFalconMsg: Current last run object is {last_run}")
+    if not last_run:
```

Comment:
i dont think there is a need for this style of init [{}... ]
on line 3240 for example you can :
last run[0] or {}